core/internal/testutils/pgtest/txdb.go

```diff
@@ -44,21 +44,30 @@ func init() {
 		// -short tests don't need a DB
 		return
 	}
-	dbURL := os.Getenv("DATABASE_URL")
-	if dbURL == "" {
-		panic("you must provide a DATABASE_URL environment variable")
+	var dbURL, which string
+	v1, v2 := os.Getenv("DATABASE_URL"), os.Getenv("CL_DATABASE_URL")
+	if v1 == "" && v2 == "" {
+		panic("you must provide a DATABASE_URL or CL_DATABASE_URL environment variable")
```

Comment:
Are we planning to deprecate `DATABASE_URL` at some point?  If so, maybe we should just mention `CL_DATABASE_URL` rather than both

Reply:
Yes, but not until 2.0. We have to support both versions of configuration during the transition release, and a particular one of the two must be used, not just either.